domains/thisiskeanyvy.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "thisiskeanyvy",
+        "email": "notifkeanyvy@gmail.com"
+    },
+
+    "record": {
+        "CNAME": ["thisiskeanyvy.pages.dev"]
```

Comment:
CNAME key must have a string value, it cannot be an array.

Reply:
OK, thank you very much!